appveyor.yml

```diff
@@ -0,0 +1,26 @@
+environment:
+  matrix:
+    - nodejs_version: '6'
+    - nodejs_version: '4'
+    - nodejs_version: '0.12'
+    - nodejs_version: '0.10'
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - set CI=true
+  - set PATH=%APPDATA%\npm;c:\MinGW\bin;%PATH%
+  - set PHANTOMJS_CDNURL=https://cnpmjs.org/downloads
```

Comment:
you'll see it's in [.travis.yml](https://github.com/mochajs/mocha/blob/master/.travis.yml) as well.  phantomjs hosts the prebuilts on bitbucket, but bitbucket has terrible rate-limiting, so CI will frequently just die when it encounters a 403.